libs/core/src/lib/popover/popover.component.ts

```diff
@@ -61,7 +61,7 @@ export class PopoverComponent {
 
     /** The element to which the popover should be appended. */
     @Input()
-    appendTo: HTMLElement | 'body';
+    appendTo: HTMLElement | 'body' = 'body';
```

Comment:
wasn't it supposed to be only for date/datetime pickers?

Reply:
Yes, i thought i had rolled this part back